### PR TITLE
add french locale

### PR DESCRIPTION
**Pull request by gorhgorh**

### CODE REVIEW

File: locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "Execute binaries from npm packages.\n%s": "Exécutez des binaire à partir de paquets de npm .\n%s",
+  "Package to be installed.": "Paquet à installer .",
```
Comment (zkat) [inline locales/fr.json:3]: Is it standard practice to put a space before `.` in French?

**Review by zkat — APPROVED** [review]
Thank you! This looks good. I have a single question about periods but otherwise I'm fine taking this as-is if you think it's ok. 🐑

File: locales/fr.json (hunk 1 above)
Comment (gorhgorh) [reply locales/fr.json:3]: I'm double checking punctuation rules in french, I just discovered that I'm writing bad french for years :) let me fix and rebase ... there is an "espace fin" in Written french, but you don't convert it to a space...

**Comment by zkat** [comment]
I'm gonna take this and do another little patch to fix up the spacing stuff real quick. Merci~

**Comment by zkat** [comment]
This was included with `npx@5.2.0`. Cheers!

**Comment by gorhgorh** [comment]
sorry... i was planning on doing this today... and thank YOU, npx is a useful tool :)

**Pull request merged**